app/controllers/ConfirmationPageController.scala

```diff
@@ -51,31 +51,37 @@ class ConfirmationPageController @Inject()(override val messagesApi: MessagesApi
     implicit request =>
 
       val result: Future[Result] = for {
-        dates: Option[CashTransactionDates] <- cache.get(request.eori)
+        dates <- cache.get(request.eori)
+        email <- customsDataStoreConnector.getEmail(request.eori)
       } yield {
-        checkDatesAndRedirect(dates)
+        email match {
+          case Right(email) =>
+            checkDatesAndEmailAndRedirect(dates, email.value)
+          case Left(_) =>
+            checkDatesAndEmailAndRedirect(dates, emptyString)
```

Comment:
Could you please check with Sujay/Harsha whether we need to show the empty string when we don't get the email then it will like below and does not look good. We can make email optional and based on that we can decide whether we need to display (on the view) the text with email or not. Let me know after having a word?

`We’ll send an email within 48 hours to when your request is processed`

Reply:
This has been resolved. Spoken with Harsha/Sujay, the whole line will be hidden if we do not retrieve email address from data-store api.